docs/build_docs/source/errors/index.rst

```diff
@@ -51,9 +51,50 @@ in your input file have correct values. Remember that ligands needs to have a un
 Connections Error
 ------------------
 
-This error indicates the the PDB file is missing the connectivity section. To resolve the issue, you should import the PDB in Schrödinger Maestro and preprocess it, launching the Protein Preparation Wizard. 
+This error indicates the the PDB file is missing the connectivity section. To resolve the issue, you should import the PDB in Schrödinger Maestro and preprocess it, launching the Protein Preparation Wizard.
 
 ..  code-block:: console
 
     pele_platform.Errors.custom_errors.ConnectionsError: Your PDB file is missing the CONECT lines. Please do not remove them after Schrodinger preprocessing.
 
+Parametrization
+------------------
+
+Sometimes parametrization of a hetero molecule (cofactor, modified residue, crystallization factor) will fail, which
+should result in the following warning message:
+
+..  code-block:: console
+
+    Failed to parametrize residue MET. You can skip it or parametrize manually
+    (see documentation: https://nostrumbiodiscovery.github.io/pele_platform/errors/index.html#parametrization).
+    The error raised was: Size of atom parameter lists should match.
+
+Remove the residue
++++++++++++++++++++++
+
+If the hetero molecule is **not necessary** to study your system, the easiest way to handle this is to **remove it from the PDB file**. Similarly, you can ignore the warning, it will not halt the simulation.
```

Comment:
Similarly, you can ignore the warning but PELE is likely going to crash because it will miss its template.